x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/components/snapshot_policies/index.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export { SnapshotPolicies } from './snapshot_policies';
+
+export interface SnapshotPoliciesProps {
```

Comment:
Typically we place the props interface for the component in the same file as the component and name it `Props`. If it needs to be exported from there I would recommend using the `as` keyword in this file: `Props as SnapshotPoliciesProps`

Reply:
You are right, I don't think this interface is needed anywhere else.